server/controllers/admin.py

```diff
@@ -255,6 +256,31 @@ def client_version(name):
                            courses=courses, current_course=current_course,
                            version=version, form=form)
 
+##########
+# Course #
+##########
+@admin.route("/course/new", methods=['GET', 'POST'])
+@is_staff()
+def create_course():
+    courses, current_course = get_courses()
+    form = forms.NewCourseForm()
+    if form.validate_on_submit():
+        new_course = Course()
+        form.timezone.data = utils.str_to_tz(form.timezone.data)
```

Comment:
Saw your note below of making the sqlalchemy time zone column be a string. I like that too and that's probably less work